builtin/logical/ssh/path_config_ca.go

```diff
@@ -104,8 +112,25 @@ func (b *backend) pathCAWrite(req *logical.Request, data *framework.FieldData) (
 		return nil, fmt.Errorf("failed to generate or parse the keys")
 	}
 
+	publicKeyEntry, err := req.Storage.Get("config/ca_public_key")
+	if err != nil {
+		return nil, fmt.Errorf("failed while reading ca_public_key: %v", err)
+	}
+	if publicKeyEntry != nil {
+		return nil, fmt.Errorf("keys are already configured; delete them before reconfiguring")
+	}
+
+	privateKeyEntry, err := req.Storage.Get("config/ca_bundle")
+	if err != nil {
+		return nil, fmt.Errorf("failed while reading ca_bundle: %v", err)
+	}
+	// Valid private key when a corresponding public key was not found
+	if privateKeyEntry != nil {
+		return nil, fmt.Errorf("configured key pair is inconsistent; delete them before reconfiguring")
+	}
+
 	err = req.Storage.Put(&logical.StorageEntry{
-		Key:   "public_key",
+		Key:   "config/ca_public_key",
```

Comment:
This will break anyone upgrading from beta but I'm okay with that, after all it's a beta :-)